front/src/assets/styles/scss/_variables.scss

```diff
@@ -289,7 +289,7 @@ $font-size-sm:                $font-size-base * .875 !default;
 $font-weight-lighter:         lighter !default;
 $font-weight-light:           300 !default;
 $font-weight-normal:          400 !default;
-$font-weight-bold:            500 !default;
+$font-weight-bold:            600 !default;
```

Comment:
500 wasn't rendering any differently than 400. `<dt>` depends on this, as well as a few other things.